examples/StreamAnalytics/streamAnalytics.bicep

```diff
@@ -0,0 +1,11 @@
+param location string = resourceGroup().location
+param name string = 'clstr${take(uniqueString(resourceGroup().id), 6)}'
+
+resource symbolicname 'Microsoft.StreamAnalytics/clusters@2020-03-01' = {
```

Comment:
Can you update `symbolicname` to something more appropriate?

Reply:
Nice catch! Updated